runtime_lib/CMakeLists.txt

```diff
@@ -81,3 +83,15 @@ install(FILES ${INSTALLS} DESTINATION ${CMAKE_INSTALL_PREFIX}/runtime_lib)
 add_subdirectory(aiesim)
 add_subdirectory(AIE)
 add_subdirectory(AIE2)
+
+if(DEFINED VITIS_ROOT)
+ExternalProject_Add(libxaie
+  PREFIX ${CMAKE_CURRENT_BINARY_DIR}/libxaieTmp
+  SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/libxaie
+  BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/libxaie
+  CMAKE_ARGS
+    -DCMAKE_MODULE_PATH=${CMAKE_MODULE_PATH}
```

Comment:
This is broken because CMAKE_MODULE_PATH can be a list, so it needs to move to CMAKE_CACHE_ARGS